doc/CustomDataContainer.md

```diff
@@ -0,0 +1,64 @@
+# CustomDataContainer
+
+
+`CustomDataContainer` renders a plug'n'play custom data card.
+
+
+## Setup
+
+Before using this component in your project, `react-i18next`, `reactstrap` and `react-hook-form` must be installed and added into the project's `package.json` file:
+
+```
+yarn add react-i18next
+yarn add reactstrap
+yarn add react-hook-form
+```
+
+At the moment works ONLY with `apiPath='seacat_auth'`
+
+
+props:
+
+
+    - `app`
+
+    - `resources`: array, of credential's resources - used for ButtonWithAuthz granting option to edit Custom data
+
+    - `customData`: object, containing data provided by api
+
+    - `setCustomData`: function managing customData's state in parent component
+
+    - `loading`: boolean, comming from the parent component deciding whether the data has already been fetched or not. if false, displays content loader
+
+    - `resource`: string, resource granting option to edit CustomData
+
+    - `uri`: string, SeaCatAuths API endpoint do save edited data. should be in format 'credentials/mongodb:JJ68' for unification, avoid passing '/credentials/mongodb:JJ68' with slash as the first character
+
+
+## Example:
+
+```
+import React from 'react';
+import { CustomDataContainer } from '../components/CustomDataContainer';
+
+const YourComponent = (props) => {
+
+        ...
+
+		return (
+            <>
+
+                ...
+
+                    <CustomDataaContainer
+						resources={resources}
```

Comment:
@petrKavulok Watch out for padding here, it is not necessary to have it that huge